### PR TITLE
Homework 8

### PR DESCRIPTION
**Pull request by PashaMisan**
Ansible 2

### CODE REVIEW

File: ansible/HW8.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+curl -I 3.127.223.10:8000 | tee output/curl.txt
+```
+
+> [Вивід](output/curl.txt)
+
+### Примітки
+
+- При генерації образу для бд, щоб не дублювати змінні плейбуків для образу та для деплою, помістив їх у
+  [vars.yml](vars.yml). Тому опцію extra-vars не використав, оскільки в ній вже не було потреби.
+- Файл [inventory](inventory) та шаблон сервісу [app.service.j2](templates/app.service.j2) створюються автоматично після
```
Comment (saaverdo) [inline ansible/HW8.md:13]: 👍🏻

File: ansible/db_install.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        state: present
+        update_cache: true
+      loop: "{{ prepare_host_packages }}"
+
+    - name: Install pymysql python package
+      ansible.builtin.pip:
+        name: pymysql
+        state: present
+
+- name: Configure MySQL
```
Comment (saaverdo) [inline ansible/db_install.yml:22]: As all plays are applied to the same hosts they could be merged into one play.

File: ansible/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        repo: https://github.com/saaverdo/flask-alb-app
+        dest: /home/ubuntu/flask-alb-app
+        version: orm
+
+    - name: Install Python requirements
+      pip:
+        requirements: /home/ubuntu/flask-alb-app/requirements.txt
+        extra_args: "--no-cache-dir"
+
+    - name: Include vars file
```
Comment (saaverdo) [inline ansible/deploy.yml:16]: It's OK, but usually vars file included at the play level.

File: ansible/HW8.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+```
+curl -I 3.127.223.10:8000 | tee output/curl.txt
+```
+
+> [Вивід](output/curl.txt)
+
+### Примітки
+
+- При генерації образу для бд, щоб не дублювати змінні плейбуків для образу та для деплою, помістив їх у
+  [vars.yml](vars.yml). Тому опцію extra-vars не використав, оскільки в ній вже не було потреби.
```
Comment (saaverdo) [inline ansible/HW8.md:12]: Good!

File: ansible/templates/app.service.j2
Comment (saaverdo) [inline ansible/templates/app.service.j2:1]: Nice solution.
Option with PreExec script updating env vars from SSM parameters store maybe preferrable, but I appreciate combining TF and Ansible logic in your solution.

**Review by saaverdo — APPROVED** [review]
Very, very good!
Approved)

File: ansible/templates/app.service.j2
Comment (PashaMisan) [reply ansible/templates/app.service.j2:1]: Thanks for the feedback! I'll remember that for next time!

**Pull request merged**